pubnub-core/src/mock/runtime.rs

```diff
@@ -29,6 +29,6 @@ impl Runtime for MockRuntime {
     where
         F: Future<Output = ()> + Send + 'static,
     {
-        self.mock_workaround_spawn(Box::pin(future))
+        self.mock_workaround_spawn(Box::pin(future));
```

Comment:
I'm still getting used to the way semicolon works in rust. From my understanding it suppresses the result of the expression from getting returned. What is happening here?

Reply:
If the last line does not end in a semicolon, the return value of the expression is also the return value of the function. If you put a semicolon, it is not used as the return value for the function. In all of these cases, the linter was complaining that we were implicitly returning a value that was not being used, and wanted semicolons to make that explicit.

Reply:
Awesome, was guessing it was the linter complaining. Thanks for the clarification.